src/transport/local.rs

```diff
@@ -89,19 +91,14 @@ impl Transport for LocalTransport {
     fn write_file(&self, relpath: &str, content: &[u8]) -> io::Result<()> {
         let full_path = self.full_path(relpath);
         let dir = full_path.parent().unwrap();
-        let mut temp = tempfile::Builder::new()
-            .prefix(crate::TMP_PREFIX)
-            .tempfile_in(dir)?;
-        if let Err(err) = temp.write_all(content) {
-            let _ = temp.close();
-            return Err(err);
-        }
-        if let Err(persist_error) = temp.persist(&full_path) {
-            persist_error.file.close()?;
-            Err(persist_error.error)
-        } else {
-            Ok(())
-        }
+        let temp = Alphanumeric.sample_string(&mut thread_rng(), 12);
```

Comment:
Why this rather than tempfile?

Reply:
It's been a while since I made this change—I'm actually not sure what this is for.